o4/o4.py

```diff
@@ -989,23 +989,26 @@ def man(cmd):
 
     quiet = '-q' if seed else ''
     retry = (f"| {manifold_big} {o4bin} drop --checksum"
+             f"{keep_case}"
              f"| {gatling_verbose} {o4bin} {quiet} {pyforce} sync -f"
              f"| {manifold_big} {o4bin} drop --checksum"
              f"| {gatling_verbose} {o4bin} {quiet} {pyforce} sync -f"
              f"| {manifold_big} {o4bin} drop --checksum"
              f"| {o4bin} fail")
 
-    syncit = f"| {gatling_verbose} {o4bin} {quiet} {pyforce} sync{force}"
+    syncit = (f"{keep_case}"
+              f"| {gatling_verbose} {o4bin} {quiet} {pyforce} sync{force}")
     if seed:
```

Comment:
Please consider making this an if/else. I know sometimes it's clear to set the normal case, then conditionally change it, but reading this it took me a while to figure that it wasn't putting keep_case in twice.

Reply:
I wanted to make a small change, since we're in the heat of things. I *HATE* how this turned out and we need to give the whole thing a make-over. I had to run through every permutation to see what the resulting cmdline was. 

It started out cute and cuddly and now it's a fully grown grizzly. :/

Reply:
Yep.